frontend/cypress/support/index.js

```diff
@@ -30,6 +30,13 @@
 // ***********************************************************
 
 import "./commands"
+const dayjs = require("dayjs")
```

Comment:
Why require and not import? Not the end of the world, but it seems different from the line above.

Reply:
No reason, just the syntax they used in the migration guide. Changed! 👍🏼